lms/views/lti/basic_launch.py

```diff
@@ -35,12 +33,10 @@ def __init__(self, context, request):
         self.assignment_service: AssignmentService = request.find_service(
             name="assignment"
         )
-        self.grouping_service: GroupingService = request.find_service(name="grouping")
-        self.course_service: CourseService = request.find_service(name="course")
+        self._guid = self.request.lti_params.get("tool_consumer_instance_guid")
+        self._resource_link_id = self.request.lti_params.get("resource_link_id")
```

Comment:
Maybe a bit silly to do this, avoid some typing :shrug: 

I was thinking that on top of the dict interface (maybe eventually replacing it?) we could dataclass-isfy LTIParams to do

`request.lti_params.guid`


Not for this PR.